pruning/config/iterators.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass
+
+from omegaconf import MISSING
+
+
+@dataclass
+class PruningIterator:
+    name: str = MISSING
+    start: float = MISSING
+    end: float = MISSING
+    step: float = MISSING
+
+
+@dataclass
+class Iterative(PruningIterator):
```

Comment:
I think that's unclear naming, there's a missing verb, maybe IterativePruning or IterativePruningStrategy etc.